### ISSUE TITLE
[Bug]: menu not visibile when task bar is on top

### ISSUE THREAD

**Issue by sn1ce** [sblantipodi/firefly_luciferin#86]
### Firefly Luciferin version

v2.7.3 + v2.8.4

### Glow Worm Luciferin version

v5.7.2

### Firmware type

FULL

### What is the stream method?

WiFi Stream

### Fiefly Luciferin config file

```shell
---
numberOfCPUThreads: 1
captureMethod: "DDUPL"
serialPort: "COM6"
baudRate: "115200"
defaultLedMatrix: "FullScreen"
autoDetectBlackBars: true
topLed: 0
leftLed: 0
rightLed: 0
bottomLeftLed: 5
bottomRightLed: 5
bottomRowLed: 26
orientation: "Clockwise"
timeout: 100
screenResX: 1440
screenResY: 2560
osScaling: 100
gamma: 2.2
whiteTemperature: 65
colorMode: 1
wifiEnable: false
mqttServer: "tcp://192.168.1.3:1883"
mqttTopic: "glowwormluciferin"
mqttUsername: ""
mqttPwd: ""
mqttEnable: false
mqttStream: false
streamType: "UDP stream"
checkForUpdates: true
eyeCare: false
nightModeFrom: "22:00"
nightModeTo: "08:00"
nightModeBrightness: "0%"
toggleLed: true
desiredFramerate: "30"
colorChooser: "255,82,0,255"
brightness: 255
ledStartOffset: 0
splitBottomMargin: "15%"
grabberAreaTopBottom: "8%"
grabberSide: "8%"
gapTypeTopBottom: "8%"
gapTypeSide: "0%"
startWithSystem: true
multiMonitor: 1
monitorNumber: 0
syncCheck: true
effect: "Bias light"
audioLoopbackGain: 0.0
audioDevice: "Default audio output (WASAPI)"
audioChannels: "2 channels"
multiScreenSingleDevice: false
powerSaving: "Disabled"
theme: "Classic theme"
language: "English"
groupBy: 1
defaultProfile: "tray.icon.default"
ledMatrix:
  FullScreen:
    1:
      x: 830
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    2:
      x: 952
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    3:
      x: 1074
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    4:
      x: 1196
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    5:
      x: 1318
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    6:
      x: 0
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    7:
      x: 122
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    8:
      x: 244
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    9:
      x: 366
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    10:
      x: 488
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
  Letterbox:
    1:
      x: 830
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    2:
      x: 952
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    3:
      x: 1074
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    4:
      x: 1196
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    5:
      x: 1318
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    6:
      x: 0
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    7:
      x: 122
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    8:
      x: 244
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    9:
      x: 366
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
    10:
      x: 488
      "y": 2049
      width: 122
      height: 146
      groupedLed: false
  Pillarbox:
    1:
      x: 830
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    2:
      x: 952
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    3:
      x: 1074
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    4:
      x: 1196
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    5:
      x: 1318
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    6:
      x: 0
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    7:
      x: 122
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    8:
      x: 244
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    9:
      x: 366
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
    10:
      x: 488
      "y": 2356
      width: 122
      height: 204
      groupedLed: false
splitBottomRow: true
extendedLog: false
configVersion: "2.7.3"
```

### How to reproduce

Hi there,

first of all, thanks for this cool repo! I think when it's working, it will be awesome!

My Issue is simple. I installed with the newest version v2.8.4 on Windows, installed Glow Worm on a Wemos D1 mini - and the UI started, after I was trying to use it.
I could do some settings and after pressing "save", the UI was gone, but Firefly is still in my tray. 

And here is the Issue -> I can't click on the Icon try to open up (i guess there should be "settings" and "quit" when rightclicking), but nothing happens.
I installed then the last version, 2.7.3, and do have the same Issue here.
I am using W10  - Version 21H1 Build 19043.

Basically I am not sure, why this is happening, any guess here?

And i guess also, in the documentation in "Compatible Hardware" under section "Simple schematic" is a small drawing issue? Since PIN2 (which is set by default) is on the Wemos D1 "D4" and not D1 as in your schematic. 

And also to mention, when I connect to the Wemos over IP, i can set Solid colors and my WS2812B are working perfectly.

If you need something, I am glad to help and provide everything you need!

Many many thanks!

**Comment by sn1ce** [comment]
Hi again,

wow, I was able to manage it! It so simple... My Taskbar is on the Top, so I was not able to see it. So maybe a feature request ?

But I also noticed, that RGB is not correct with my stripes, i guess its GRB.

Cheers!

**Comment by sblantipodi** [comment]
thank you for reporting the issue @sn1ce, I'll look into it.
what kind of strips are you using?

> But I also noticed, that RGB is not correct with my stripes, i guess its GRB.

can you elaborate this please? WS2812B is fully supported you should not have problems, but what is the problem you are experiecing exactly?

thanks
Davide
